File: src/main/webapp/assets/js/Validator.js
class Validator{
	
	static isNull(value){

	return `${value}`.trim().length === 0  || value === null;
	

	}
	
	static isEmail(email) {
	       if(`${email}`.match(/^[\w\.-]+@[a-zA-Z\d\.-]+\.[a-zA-Z]{2,}$/)){
	           return true;
	       }
	       return false;
	   }

	   static clear(text){
	   	
	   	text = `${text}`;
	   	text = text.replace(/(<[!\/]?\w*(\s\w+(=["`][^"`]*["`])?)*\s?>|([`";]+))/,"");
	   	text = text.replace(/\b(select|drop|insert|delete|update|where|table|from)\b|['";\\-]/gi,"");
	   	return text;

	   }
	   
	  
	   
	
	
} 

File: src/java/programLibraries/api/Validator.java
package programLibraries.api;

public class Validator {

    public static boolean isEmail(String value) {
        return value.matches("^[\\w.%+-]+@[\\w.-]+\\.[a-zA-Z]{2,}$");
    }

    public static String clear(String value) {
        if (value == null) {
            return "";
        }

        value = value.replaceAll("\\b(?i)(select|drop|insert|delete|update|where|table|from)\\b|['\";\\-]", "");
        value = value.replaceAll("<[^>]*>|[`\":;]+", "");

        return value;
    }
}